example.js

```diff
@@ -1,53 +1,50 @@
-const Lammas = require('./lammas');
+import Sheep from './sheep';
```

Comment:
this is fine, just adding comment that eslint failing here...

require and import in same project is not the best, will explain more in next lesson when we do more deep drive into linting...